include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2577,6 +2580,48 @@ int bt_br_discovery_start(const struct bt_br_discovery_param *param,
  */
 int bt_br_discovery_stop(void);
 
+struct bt_br_discovery_cb {
```

Comment:
All these Classic-related APIs should probably be moved under `include/zephyr/bluetooth/classic/`

Reply:
..and I think it'd be good to do that before extending the current API

Reply:
Updated